src/managers/KeybindManager.cpp

```diff
@@ -421,10 +421,11 @@ bool CKeybindManager::handleKeybinds(const uint32_t& modmask, const std::string&
         Debug::log(LOG, "Keybind handling only locked (inhibitor)");
 
     for (auto& k : m_lKeybinds) {
+        std::string currentSubmapName = m_szCurrentSelectedSubmap == nullptr ? "" : m_szCurrentSelectedSubmap->getName();
```

Comment:
`== nullptr` redundant

Reply:
I assume you wanted it move out of the for

Reply:
no I want it to be `!m_szCurrentSelectedSubmap`